workers/data_refinery_workers/processors/create_compendia.py

```diff
@@ -49,6 +53,12 @@ def log_state(message, job_id, start_time=False):
 def _prepare_input(job_context: Dict) -> Dict:
     start_time = log_state("prepare input", job_context["job"].id)
 
+    job_context["primary_organism"] = max(job_context["samples"],
+                                          key=lambda organism:len(job_context["samples"][organism]))
+    job_context["all_organisms"] = job_context["samples"].keys()
+    all_samples = itertools.chain(*job_context["samples"].values())
```

Comment:
should we convert this to a list? I think it returns an iterator

Reply:
I have tested this and it works. So it's a matter of preference

Reply:
evaluating it now because we check it at annotations step